app/controllers/concerns/exception_handler.rb

```diff
@@ -0,0 +1,28 @@
+module ExceptionHandler
+  extend ActiveSupport::Concern
+
+  class AuthenticationError < StandardError; end
+  class MissingToken < StandardError; end
+  class InvalidToken < StandardError; end
+
+  included do 
+    rescue_from ActiveRecord::RecordInvalid, with: :four_twenty_two
+    rescue_from ExceptionHandler::AuthenticationError, with: :unauthorized_request
+    rescue_from ExceptionHandler::MissingToken, with: :four_twenty_two
```

Comment:
Could consider other error codes for tokens. 422 is used when the incoming params has issues. Consider 401/403